Dockerfile.ubuntu

```diff
@@ -43,11 +44,18 @@ RUN apt-get install -y cmake libboost-python-dev python-dev
 # ZBar
 RUN apt-get install -y libzbar-dev python-dev
 
+# pip (need a recent version e.g. for wheels support - ubuntu is obsolete)
+RUN wget https://bootstrap.pypa.io/get-pip.py
+RUN python get-pip.py
```

Comment:
I believe a `sudo pip install -U pip` should work here (That's what I had tried on Centos)